doc/lua_api.txt

```diff
@@ -7538,11 +7548,15 @@ Player properties need to be saved manually.
         collide_with_objects = true,
         -- Collide with other objects if physical = true
 
-        collisionbox = {-0.5, -0.5, -0.5, 0.5, 0.5, 0.5},
-        selectionbox = {-0.5, -0.5, -0.5, 0.5, 0.5, 0.5},
-        -- Selection box uses collision box dimensions when not set.
-        -- For both boxes: {xmin, ymin, zmin, xmax, ymax, zmax} in nodes from
-        -- object position.
+		-- For both boxes: {xmin, ymin, zmin, xmax, ymax, zmax} in nodes from
+		-- object position.
+        collisionbox = {-0.5, 0.0, -0.5, 0.5, 0.5, 0.5},  -- Default
+        selectionbox = {rotate = false; -0.5, 0.0, -0.5, 0.5, 0.5, 0.5},
+        -- Selection box uses collision box *dimensions* when not set.
+        -- Setting `rotate = true` in the `collisionbox` has no effect.
+        -- If `rotate = true`, the box will be rotated by the object's rotation
+        -- (plus the attachment rotation on clients)
```

Comment:
As someone who's not read the code, this is rather confusing.

> Selection box uses collision box *dimensions* when not set.

Specifying, and then emphasizing 'dimensions' is odd, what else is there to copy?

> Setting `rotate = true` in the `collisionbox` has no effect.

Oh, so `rotate` isn't copied from the collision box table?
Right, that's what the previous sentence meant.

> If `rotate = true`, the box will be rotated by the object's rotation (plus the attachment rotation on clients).

*rotated by the object's rotation*…

Assuming I'm correct:
```suggestion
        -- Collision boxes cannot rotate, setting `rotate = true` on it has no effect.
        -- If not set, the selection box copies the collision box, and will also not rotate.
        -- If `rotate = false`, the selection box will not rotate with the object itself, remaining fixed to the axes.
        -- If `rotate = true`, it will match the object's rotation and any attachment rotations.
        -- Raycasts use the selection box and object's rotation, but do *not* obey attachment rotations.
```
(hard wrap *:(* to taste)

Reply:
Thanks for rephrasing.

> (plus the attachment rotation on clients)

was supposed to emphasize that the server is unaware of the attachment rotation and will be unable to take it into account when raycasting; this is redundant with the note I added on raycasts however:

> Raycasts don't always work properly for attached objects as the server has no knowledge of models & bones.

Reply:
Redundancy is good here IMO, as the two sections are reasonably separated.
Also works as a little hint to ray-casts being a relevant feature alongside this, previously there was no reference to them at all.